gravitee-apim-console-webui/src/management/settings/groups/group/add-members-dialog/add-members-dialog.component.ts

```diff
@@ -104,8 +104,11 @@ export class AddMembersDialogComponent implements OnInit {
 
   private initializeForm() {
     this.addMemberForm = new FormGroup({
-      defaultAPIRole: new FormControl({ value: 'USER', disabled: false }),
-      defaultApplicationRole: new FormControl<string>({ value: 'USER', disabled: false }),
+      defaultAPIRole: new FormControl({ value: this.data.group.roles['API'] ? this.data.group.roles['API'] : 'USER', disabled: false }),
```

Comment:
```suggestion
      defaultAPIRole: new FormControl({ value: this.data.group.roles['API']  ??  'USER', disabled: false }),
```

I may be wrong but it's shorter and it should do the same I think 🤔

Reply:
Done